### ISSUE TITLE
Fluent Dark doesn't render the checkboxes correctly

### ISSUE THREAD

**Issue by stantios** [PrismLauncher/Themes#12]
I'm not sure if it's supposed to be like this but in my opinion, I think there should be a blue background with a tick logo inside. This is what it looks like for now: 
![image](https://user-images.githubusercontent.com/98621617/202897179-2d08ac61-e156-4144-b788-17caadd97097.png)

**Comment by DioEgizio** [comment]
restart the launcher, idk why it doesn't work on first launch

**Comment by TayouVR** [comment]
i've seen that too, am not a theme dev though, so not sure how to fix it. Its just how the theme is. Maybe someone with actual qss skills can look through the fluent qss file and find the culprit.

**Comment by DioEgizio** [comment]
Maybe it's just yet another [QSS bug](https://www.kdab.com/say-no-to-qt-style-sheets/)

**Comment by TayouVR** [comment]
I think this was fixed in some prior PR, can people confirm that?

**Comment by stantios** [comment]
Yep, can confirm that the theme is working and everything is rendering correctly.
